app/app.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express')
+const app = express()
+const { getTopics } = require('./controllers/topics-controller')
+
+app.use(express.json())
+
+app.get('/api/topics', getTopics)
+
+//ERROR HANDLING
+app.use((err, req, res, next) =>{
+    //psql errors
+    const psqlErrCodes = {}
+
+    if(Object.keys(psqlErrCodes).includes(err.code)){
+        response.status(400).send({msg: "Bad request"})
+    }
+})
+
+app.listen(9090, ()=>{
+    console.log('listening on 9090')
+})
```

Comment:
This is nice to see, but should be in a separate file instead of being within app.js